src/main/java/core/utils/StringToDateConverter.java

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+
+package core.utils;
+
+import org.openqa.selenium.InvalidArgumentException;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+
+public class StringToDateConverter {
+    private Calendar calendar = Calendar.getInstance();
+
+    /**
+     * Converts a provided String to Date.
+     *
+     * @param string with the date on format like:
+     *               TODAY, YESTERDAY, TOMORROW, or
+     *               [number] [time unit] ago, or
+     *               [number] [time unit] from now, or
+     *               MM/DD/YYYY
+     * @return the date in Date type
+     */
+    public Date convertStringToDate(final String string) throws ParseException {
+        if (string == null | string.equals("")) {
```

Comment:
i'm not shure about this operator | , i think you want to use an OR ||

Reply:
Done.